acstools/acsphotcte.py

```diff
@@ -0,0 +1,289 @@
+        with native python types (int, float, list, str). Hence, before
+        we serialize the inputs as a JSON object we need to compare all the
+        input data types to their python counterparts.
+
+        Parameters
+        ----------
+        inputs : dict
+            Dictionary containing all of the user supplied inputs
+
+        Returns
+        -------
+        inputs : dict
+            The original dictionary with all of the inputs converted to their
+            python counterparts for JSON serialization
+        """
+        for key in inputs.keys():
+            # Check if the input is Iterable and not a list  (e.g. np.array)
+            # We also need to make sure the input is not a str, since str are
+            # Iterable in Python.
+            if isinstance(inputs[key], Iterable) and \
+                    not isinstance(inputs[key], list) and \
+                    not isinstance(inputs[key], str):
+                # Convert the array to a python list
+                inputs[key] = list(inputs[key])
+
+                # Check if the first element of the input array is a float
+                if not isinstance(inputs[key][0], float):
+                    LOG.info('Converting elements..')
+                # Convert the array of elements to floats
+                    inputs[key] = list(map(float, inputs[key]))
+
+            # Check if the input is a python float, not an np.float32
+            # Handles the case of str inputs as well
+            elif not isinstance(inputs[key], float):
+                try:
+                    inputs[key] = float(inputs[key])
+                except TypeError as e:
+                    # TypeError during the conversion, print the info and
+                    # return the key it failed on
+                    LOG.info(e)
+                    return key
+                else:
+                    if key == 'radius' and inputs[key] not in [3., 5.]:
+                        LOG.info('Submitted radii not supported...')
+                        return key
+        return inputs
+
+    def _compute_corrections(
+            self,
+            radius=None,
+            ytransfers=None,
+            mjd=None,
+            local_skys=None,
+            fluxes=None,
+            api_id=None,
+            api_key=None,
+    ):
+        """Compute the photometric CTE corrections
+
+        Parameters
+        ----------
+        radius : {3, 5}
+            Aperture radius used to extract photometry
+
+        ytransfers : float, list
+            Number of y transfers. For chip 2 on WFC this is just the Y value,
+            for chip 1 it is 2048 - Y value.
+
+        mjd : float
+            MJD date of the observation
+
+        local_sky : float, list
+            Local sky background for each source
+
+        fluxes : float, list
+            Computed source flux, in electrons, for 3 or 5 pixel apertures.
+
+        Returns
+        -------
+
```

Comment:
What does it return?

Reply:
I'm going to just remove this method as it's not really necessary.